admin/routes/completed-check.js

```diff
@@ -1,10 +1,19 @@
 const express = require('express')
 const router = express.Router()
+const moment = require('moment')
 const { postCheck } = require('../controllers/completed-check')
+const winstonLogger = require('./../winstonLogger')
 
 router.route('/').all((req, res) => {
-  if (req.method !== 'POST') return res.sendStatus(405)
+  const apiLogger = winstonLogger.apiLogger()
+  apiLogger.info(`MTC-API: completed-started (${moment()}) reached`)
```

Comment:
rather than a static text of `completed-started` please use `req.url`

Reply:
OK, I'll change it. The content logged is totally temporary, please let me know if you prefer a different format, etc.